### PR TITLE
feature(core): close vault screen 2.0

### PR DESCRIPTION
**Pull request by pierregee**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+    rhsThemedProps: {
+      light: tailwind("text-mono-light-v2-900"),
+      dark: tailwind("text-mono-dark-v2-900"),
+    },
+  };
+  return (
+    <>
+      <ThemedViewV2
+        style={tailwind("p-5 border-0.5 mt-6 w-full rounded-lg-v2")}
+        dark={tailwind("border-gray-300")}
```
Comment (chloezxyy) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx:148]: dark={tailwind("border-mono-dark-v2-300")}

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+      <View style={tailwind("items-center")}>
+        <Image
+          source={CloseVaultImg}
+          style={[
+            tailwind("items-center justify-center"),
+            { width: 122, height: 95 },
+          ]}
+          resizeMode="contain"
+        />
+        <ThemedTextV2 style={tailwind("mb-1 font-normal-v2 text-sm mt-6")}>
```
Comment (JoshuaC817) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx:87]:
```suggestion
        <ThemedTextV2 style={tailwind("mb-1 font-normal-v2 text-base mt-6")}>
```

**Comment by JoshuaC817** [comment]
image
Wallet with short address label not aligned to right

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+      light: tailwind("text-mono-light-v2-900"),
+      dark: tailwind("text-mono-dark-v2-900"),
+    },
+  };
+  return (
+    <>
+      <ThemedViewV2
+        style={tailwind("p-5 border-0.5 mt-6 w-full rounded-lg-v2")}
+        dark={tailwind("border-gray-300")}
+        light={tailwind("border-gray-300")}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx:149]:
```suggestion
        light={tailwind("border-mono-light-v2-300")}
```

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+            "screens/CloseVaultScreen",
+            "All remaining collaterals will be returned to your wallet."
+          )}
+        </ThemedTextV2>
+        <SubmitButtonGroupV2
+          isDisabled={hasPendingJob || hasPendingBroadcastJob}
+          label={translate("screens/CloseVaultScreen", "Close vault")}
+          displayCancelBtn={false}
+          onSubmit={onSubmit}
+          buttonStyle="mt-4 mx-7"
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx:121]:
```suggestion
          buttonStyle="mt-5 mx-7"
```

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+            "Are you sure you want to close your vault"
+          )}
+        </ThemedTextV2>
+        <ThemedTextV2
+          style={tailwind("font-normal-v2 text-sm mt-1 w-56")}
+          numberOfLines={1}
+          ellipsizeMode="middle"
+          testID="edit_loan_scheme_vault_id"
+        >
+          {`${vaultId}?`}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx:99]: can use the same way to display inline truncated vaultId at line 264

File: mobile-app/app/components/SubmitButtonGroupV2.tsx [hunk 6]
```diff
@@ -76,17 +76,6 @@ export function SubmitButtonGroupV2({
     }
   };
 
-  // const getSubmittingLabel = (): string | undefined => {
```
Comment (kyleleow) [inline mobile-app/app/components/SubmitButtonGroupV2.tsx:79]: Maybe add comment to describe why

Comment (pierregee) [reply mobile-app/app/components/SubmitButtonGroupV2.tsx:79]: Oh I removed it cos it's not used anymore

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx [hunk 7]
```diff
@@ -0,0 +1,304 @@
...
+      light: tailwind("text-mono-light-v2-500"),
+      dark: tailwind("text-mono-dark-v2-500"),
+    },
+    rhsThemedProps: {
+      light: tailwind("text-mono-light-v2-900"),
+      dark: tailwind("text-mono-dark-v2-900"),
+    },
+  };
+  return (
+    <>
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/CloseVaultScreenV2.tsx:145]: i think we can also remove this

File: mobile-app/app/components/SubmitButtonGroupV2.tsx (hunk 6 above)
Comment (kyleleow) [reply mobile-app/app/components/SubmitButtonGroupV2.tsx:79]: Ops! sorry i thought it was commented

**Pull request merged**